ethereum_history_api/oracles/src/noir/oracles/common/decode.ts

```diff
@@ -23,7 +23,7 @@ export function decodeAddress(arg: string[]): Address {
     assert(0 <= d && d <= MAX_U8 && isHex(e), `Invalid address, with byte: ${e}`);
   }
 
-  const result = decodeHexValue(arg);
+  const result = getAddress(decodeHexValue(arg));
```

Comment:
Why not use `decodeAddress` instead?

Reply:
Can you be more specific here? Which function should I use here?